Sporadic test failure in ReadWriteTest.readWriteTest

**Problem description**
Found test `io.pravega.test.integration.ReadWriteTest.readWriteTest` failing due to:
```
java.util.concurrent.ExecutionException: java.lang.InterruptedException
	at java.base/java.util.concurrent.CompletableFuture.reportGet(CompletableFuture.java:395)
	at java.base/java.util.concurrent.CompletableFuture.get(CompletableFuture.java:1999)
	at io.pravega.test.integration.ReadWriteTest.readWriteTest(ReadWriteTest.java:211)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:298)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:292)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.lang.Thread.run(Thread.java:834)
Caused by: java.lang.InterruptedException
	at java.base/java.util.concurrent.CompletableFuture.reportGet(CompletableFuture.java:385)
	at java.base/java.util.concurrent.CompletableFuture.get(CompletableFuture.java:1999)
	at io.pravega.common.concurrent.Futures.getThrowingException(Futures.java:178)
	at io.pravega.client.segment.impl.SegmentMetadataClientImpl.lambda$getSegmentInfo$13(SegmentMetadataClientImpl.java:239)
	at io.pravega.common.util.Retry$RetryAndThrowBase.run(Retry.java:173)
	at io.pravega.client.segment.impl.SegmentMetadataClientImpl.getSegmentInfo(SegmentMetadataClientImpl.java:239)
	at io.pravega.client.state.impl.RevisionedStreamClientImpl.readFrom(RevisionedStreamClientImpl.java:134)
	at io.pravega.client.state.impl.StateSynchronizerImpl.fetchUpdates(StateSynchronizerImpl.java:82)
	at io.pravega.client.stream.impl.ReaderGroupStateManager.readerShutdown(ReaderGroupStateManager.java:155)
	at io.pravega.client.stream.impl.ReaderGroupStateManager.readerShutdown(ReaderGroupStateManager.java:147)
	at io.pravega.client.stream.impl.EventStreamReaderImpl.closeAt(EventStreamReaderImpl.java:414)
	at io.pravega.client.stream.impl.EventStreamReaderImpl.close(EventStreamReaderImpl.java:402)
	at io.pravega.test.integration.ReadWriteTest.lambda$startNewReader$1(ReadWriteTest.java:259)
	at java.base/java.util.concurrent.CompletableFuture$AsyncRun.run(CompletableFuture.java:1736)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	... 1 more
```

**Problem location**
`io.pravega.test.integration.ReadWriteTest.readWriteTest`

**Suggestions for an improvement**
Make the test to pass reliably.

--- a/test/integration/src/test/java/io/pravega/test/integration/ReadWriteTest.java
+++ b/test/integration/src/test/java/io/pravega/test/integration/ReadWriteTest.java
@@ -67,8 +67,8 @@ public class ReadWriteTest {
     private static final String STREAM_NAME = "testMultiReaderWriterStream" + RandomFactory.create().nextInt(Integer.MAX_VALUE);
     private static final int NUM_WRITERS = 20;
     private static final int NUM_READERS = 20;
-    private static final long TOTAL_NUM_EVENTS = 20000;
-    private static final int NUM_EVENTS_BY_WRITER = 1000;
+    private static final int NUM_EVENTS_BY_WRITER = 500;
+    private static final long TOTAL_NUM_EVENTS = NUM_WRITERS * NUM_EVENTS_BY_WRITER;
     private AtomicLong eventData;
     private AtomicLong eventReadCount;
     private AtomicBoolean stopReadFlag;
@@ -242,7 +242,7 @@ public class ReadWriteTest {
                     EventWriterConfig.builder().build());
             for (int i = 0; i < NUM_EVENTS_BY_WRITER; i++) {
                 long value = data.incrementAndGet();
-                log.info("Writing event {}", value);
+                log.debug("Writing event {}", value);
                 writer.writeEvent(String.valueOf(value), value);
                 writer.flush();
             }
@@ -263,7 +263,7 @@ public class ReadWriteTest {
                     ReaderConfig.builder().build());
             while (!(exitFlag.get() && readCount.get() == writeCount.get())) {
                 final Long longEvent = reader.readNextEvent(SECONDS.toMillis(2)).getEvent();
-                log.info("Reading event {}", longEvent);
+                log.debug("Reading event {}", longEvent);
                 if (longEvent != null) {
                     //update if event read is not null.
                     readResult.add(longEvent);

